### DISCUSSION TITLE
Usage docmentation of subprotocols (MQTT / WebSocket)

### DISCUSSION THREAD

**Discussion by cayolblake** [walkor/Workerman#807] [Q&A]
Hey @walkor 

I was going through the commits and found https://github.com/walkor/workerman/commit/796c413d0ef42b44da2e9ff6756241ab76c90b9c and its pull request which is basically the only mention of WebSocket subprotocols.

Is it possible to have an example in the project's main README.md to showcase its usage?

**Comment by walkor** [comment]
It is not an important feature and may change later. So it will not appear in README. And the author has shown an example in pr.
